habitat-lab/habitat/config/default_structured_configs.py

```diff
@@ -155,13 +155,74 @@ class TeleportActionConfig(ActionConfig):
 @attr.s(auto_attribs=True, slots=True)
 class VelocityControlActionConfig(ActionConfig):
     type: str = "VelocityAction"
-    # meters/sec
-    lin_vel_range: List[float] = [0.0, 0.25]
-    # deg/sec
-    ang_vel_range: List[float] = [-10.0, 10.0]
+    lin_vel_range: List[float] = [0.0, 0.25]  # meters/sec
+    ang_vel_range: List[float] = [-10.0, 10.0]  # deg/sec
```

Comment:
If we are using radian inputs, lets also use radians here to prevent any possible errors